server/sql/create_puzzles.sql

```diff
@@ -10,7 +10,7 @@ IF NOT EXISTS puzzles
   uploaded_at timestamp without time zone,
 
   -- static properties of the puzzle
-  content json
+  content jsonb
```

Comment:
This makes sense, thanks for this change. I'm on board with using jsonb since 
1) Seems more flexible and simpler than either extracting or replicating data as string columns
2) Performant enough, considering the small size of the puzzles table.

Reply:
i'll apply this in a separate PR. i think it'll look like this: ```
ALTER TABLE table_with_json
  ALTER COLUMN my_json
  SET DATA TYPE jsonb
  USING my_json::jsonb;
```